frontend/src/components/form/password-field.tsx

```diff
@@ -0,0 +1,33 @@
+import { ViewIcon, ViewOffIcon } from '@chakra-ui/icons';
```

Comment:
Some time ago, we agreed to use MD icons from `react-icons` for consistency. There is not a rule for that, because it's not fully migrated yet. I'm actually surprised we still have this package.

Reply:
Fixed.